roles/mesos/tasks/RedHat.yml

```diff
@@ -6,3 +6,21 @@
   yum: name={{item}} state=installed
   with_items:
     - mesos-{{mesos_version}}
+
+# The Mesos package installs both the master and agent. Make the Mesos services
+# match those of the roles.
+
+- name: Check Mesos Master running
```

Comment:
I think the declarative way of doing this by using `service: name=... state=stopped` is much cleaner. Let's keep the rules in `roles/mesos/tasks/main.yml`